### PR TITLE
Upgrade to /v2 Go module

### PR DESCRIPTION
**Pull request by hownowstephen**
Proper v2 release of `go-customerio` using module pathing

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -7,14 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.10', '1.11', '1.12' ]
+        go: [ '1.15', '1.16.4' ]
```
Comment (jatinn) [inline .github/workflows/main.yml:10]: any reason to explicitly call out the patch version in `1.16.4`?

File: options.go [hunk 2]
```diff
@@ -8,28 +8,28 @@ type option struct {
 }
 
 type region struct {
-	apiURL   string
-	trackURL string
+	ApiURL   string
```
Comment (jatinn) [inline options.go:11]: guessing exported for tests?

**Review by jatinn — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (hownowstephen) [reply .github/workflows/main.yml:10]: none, aside from it's what I use locally. I'll switch it to just 1.16

File: options.go (hunk 2 above)
Comment (hownowstephen) [reply options.go:11]: yep

**Pull request merged**